src/services/AuthService.ts

```diff
@@ -31,4 +78,25 @@ export class AuthService {
       return userRepository.deleteUser(user);
     });
   }
+
+  public async getSessionsByUserId(id: Uuid): Promise<APIUserSession[]> {
+    return this.transactions.readOnly(async (transactionalEntityManager) => {
+      const sessionRepository = Repositories.session(transactionalEntityManager);
+      const sessions = await sessionRepository.getSessionsByUserId(id);
+      if (!sessions) throw new NotFoundError('User not found!');
+      const apiSessions: APIUserSession[] = [];
+      sessions.forEach(function (session) {
```

Comment:
I see this is where you're allowing multiple sessions, I'm wondering why we can't just convert sessions into that list format (would casting it just work?) but this works too